feature-group-definitions/stable-array-sort.yml

```
@@ -0,0 +1,16 @@
+name: Stable array sort
+description: Stable Array sort() function
+spec: https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.sort
+status:
+  baseline: high
+  baseline_low_date: 2018-10-16
```

Isn't Edge 79 the keystone release?

```suggestion
  baseline_low_date: 2020-01-15
```